src/components/NewsFeed/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useContext } from 'react'
```

Comment:
why 4 spaces as tab ?  we format 2 everywhere, please fix this over all PR (if there're any other places having wrong tabulation)